plugins/frontend/backstage-plugin-jira/config.d.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** Configuration for the Jira plugin */
+export interface Config {
+  jira?: {
+    /**
+     * The proxy path for Jira.
+     * Should be used if proxy is in use for security etc purposes.
+     * @visibility frontend
+     */
+    proxyPath?: string;
```

Comment:
I am not sure I follow why this proxyPath should be treated different and handled here instead of app-config?

Reply:
What do you mean by handled in app-config ? 

The addition of this file to the plugin will make sure we can configure the plugin in the app-config at the `jira` key.

Reply:
I see, what is the value of having custom proxy path instead of predefined (jira/api) one?

Reply:
I found the original issue https://github.com/RoadieHQ/roadie-backstage-plugins/issues/502#issuecomment-1113670104, will check that first

Reply:
Yes, I think the original issue was due 'target' in 'jira/api' proxy probably being misconfigured or wrong. I don't mind adding this possibility if you want but can we then add an explanation In README?